components/lcd/esp_lcd_ili9881c/esp_lcd_ili9881c.c

```diff
@@ -397,6 +387,16 @@ static esp_err_t panel_ili9881c_init(esp_lcd_panel_t *panel)
         return ESP_ERR_INVALID_ARG;
     }
 
+    // The ID register is on the CMD_Page 1
+    uint8_t ID1, ID2, ID3;
+    esp_lcd_panel_io_tx_param(io, ILI9881C_CMD_CNDBKxSEL, (uint8_t[]) {
+        ILI9881C_CMD_BKxSEL_BYTE0, ILI9881C_CMD_BKxSEL_BYTE1, ILI9881C_CMD_BKxSEL_BYTE2_PAGE1
+    }, 3);
+    esp_lcd_panel_io_rx_param(io, 0x00, &ID1, 1);
+    esp_lcd_panel_io_rx_param(io, 0x01, &ID2, 1);
+    esp_lcd_panel_io_rx_param(io, 0x02, &ID3, 1);
+    ESP_LOGI(TAG, "ID1: 0x%x, ID2: 0x%x, ID3: 0x%x", ID1, ID2, ID3);
+
     // back to CMD_Page 1
```

Comment:
Looks like this comment and the following `esp_lcd_panel_io_tx_param` is not needed anymore? Since we are already on CMD_Page 1...

Reply:
Yes, the comment and the code for navigating to the first page are no longer needed. Thanks for pointing out this oversight; the issue has been fixed.